### ISSUE TITLE
util/fix_format.sh sources a script relative to its own parent directory, not relative to its own location

### ISSUE THREAD

**Issue by erlehmann** [minetest/minetest#12568]
##### Minetest version
Commit ccdaf5de54108990fcdeb0b0ff4a4fc4cd998522 and newer

##### OS / Hardware
It does not matter.

##### Summary
The `util/fix_format.sh` script is sourcing a file relative to the root of the repository, not relative to its own location.
I think this means it can only work when executed from the root of the directory (which is not documented at all).
It also confuses the shellcheck linter when run with the `-x` option, as it can not find the sourced file.

Here is `util/fix_format.sh` in its entirety:

```
#!/bin/bash -e

. ./util/ci/clang-format.sh

fix_format
```

##### Steps to reproduce
1. Open a shell to the `util` directory.
2. Run `shellcheck -x fix_format.sh`.
3. Get an error message (SC1091).

**Comment by JosiahWI** [comment]
```
./fix_format.sh: line 3: ./util/ci/clang-format.sh: No such file or directory
```

Confirmed.

**Comment by rubenwardy** [comment]
This was intentional as I run it from the root directory like `./util/fix_format.sh`. It should probably get the script path and then the src directory from that instead, allowing it to be executed from anywhere

**Comment by relzp** [comment]
fixed(?) by da832a295e0cf5620ad89415a8ed3ddd91be1547?

**Comment by SmallJoker** [comment]
Yes. Fixed by removing the file. This issue is now obsolete.
